plugins/azure-devops/src/hooks/useBuildRuns.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BuildRun,
+  BuildRunOptions,
+} from '@backstage/plugin-azure-devops-common';
+
+import { AZURE_DEVOPS_DEFAULT_TOP } from '../constants';
+import { azureDevOpsApiRef } from '../api';
+import { useApi } from '@backstage/core-plugin-api';
+import { useAsync } from 'react-use';
```

Comment:
This is what's causing the type issues, needs to be `import useAsync from 'react-use/lib/useAsync'` ._.

It's because `react-use` is unmaintained and doesn't support TypeScript 4.5 properly. We'll be looking at options, but for now we're just avoiding the full import. Will try to set up linting for this asap but that hinges on some other refactoring first 😓

Reply:
Thanks @Rugvip will get on that now 👍

Reply:
Ok, fix pushed, just waiting for the build to go green now :)